project.clj

```diff
@@ -76,7 +76,8 @@
                  [honeysql "0.5.2"]
                  [org.clojure/data.xml "0.0.8"]
                  [com.rpl/specter "0.5.7"]
-                 [org.clojure/core.async "0.1.346.0-17112a-alpha"]]
+                 [org.clojure/core.async "0.1.346.0-17112a-alpha"]
+                 [puppetlabs/http-client "0.4.4"]]
```

Comment:
yeah, probably. the api is the same for common cases. certainly it's easy enough to do for tests.